src/pages/LandingPage.js

```diff
@@ -86,6 +90,23 @@ const LandingPage = () => {
       }, ['See this article']),
       ' for a summary of available resources.'
     ]),
+    isDataBrowserVisible() && div({
+      style: {
+        backgroundColor: colors.primary(),
+        backgroundImage: `url(${terraHero})`, backgroundSize: 'cover', borderRadius: 5,
+        display: 'flex', flexDirection: 'row', justifyContent: 'space-between', alignItems: 'center',
+        boxShadow: '0 2px 5px 0 rgba(0,0,0,0.35), 0 3px 2px 0 rgba(0,0,0,0.12)',
+        color: 'white', padding: '2rem 1rem',
+        width: 'calc(675px + 2rem)', // 3 card widths + 2 card margins to line up with the content directly above
+        marginTop: 15
+      }
+    }, [
+      h2({ style: { fontSize: 18, fontWeight: 500, lineHeight: '22px', margin: 0 } }, ['Preview the BETA Data Catalog and provide valuable feedback.']),
+      h(ButtonOutline, {
+        style: { marginLeft: '2rem', padding: '1.5rem 1rem', textTransform: 'none' },
+        onClick: () => { Nav.goToPath('library-browser') }
+      }, ['Preview BETA Data Catalog'])
```

Comment:
Is there a way to reword this so that the phrase "Preview the BETA Data Catalog" isn't repeated?

To be more consistent with the COVID banner above it, maybe something like "BETA Data Catalog" as the heading and below that in normal text, "Preview the Data Catalog and provide valuable feedback." with "Preview the Data Catalog" or the whole sentence linked to the catalog?

Reply:
I like this, going to make that change! :)